src/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+add_subdirectory(platform)
```

Comment:
Please use the imperative mood in commit message. Read more here:

https://chris.beams.io/posts/git-commit/#imperative

I would also prefer to split this in two separate patches. One patch per change.

So, you will have something like this:

kconfig: Make number of cores configurable
kconfig: Make memory limits configurable

Reply:
Commit message - sure my bad

I cannot split it since it is interconnected and would have to do workarounds for one patch to remove theme in the other.

Reply:
I'm pretty sure it can be done. 

1) 2 patches to introduce the Kconfigs (core/memory)
2) 2 patches (1 for core numbers / 1 for memory limits) that remove the hardcoded values and replace them with CONFIG_ introduced in patches below
3) patch that introduce ICELAKE stuff. Not sure about this because I need to see it first as a separate patch.

Reply:
@dbaluta When you have comment regarding something that is not in diff (f.e. commit messages), I think it's better to have it in overall review comment instead of binding it to unrelated line.

Reply:
@jajanusz sure I'm still confused sometimes about github workflow. I honestly miss review on email :)

Reply:
I may agree with intent. However You are suggesting splitting up  something that is essentially one feature. Platforms are defined by core number and available memory. One configurable without the other makes no sense to me.